notify/impl.go

```diff
@@ -660,6 +661,20 @@ func (n *Slack) Notify(ctx context.Context, as ...*types.Alert) (bool, error) {
 		Color:     tmplText(n.conf.Color),
 		MrkdwnIn:  []string{"fallback", "pretext", "text"},
 	}
+
+	var numFields = len(n.conf.Fields)
+	if numFields > 0 {
+		var fields = make([]slackAttachmentField, numFields)
+		for k, v := range n.conf.Fields {
+			fields[k] = slackAttachmentField{
+				v["title"],
```

Comment:
let's add `tmplText()` around both `title` and `value`, so the users can optionally define a template